ReleaseProcessAutomation.UnitTests/Config/ConfigReaderTests.cs

```diff
@@ -75,4 +76,75 @@ public void GetConfigPathFromBuildProject_InCurrentWorkingDir_GetsProperName ()
 
     Assert.That(configPath, Is.EqualTo("Build/Customizations/ReleaseProcessScript.Test.config"));
   }
+
+  [Test]
```

Comment:
Do we have a test that asserts the behavior that no MSBuild exe is found? If not we should add one

Reply:
Im questioning whether we need a test to figure out whether a proper MSBuild exe is found, or just whether something was added in the config that should point to some existent path. 

I already implemented the second one, but i wouldnt know whether or how to do the proper MSBuild.exe testing other than looking at whether the last part of the file path is called msbuild.exe